index.html

```diff
@@ -4201,7 +4205,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<div class="role-description">
 				<p>An option in a set of choices contained by a <rref>menu</rref> or <rref>menubar</rref>.</p>
 				<p>Authors MAY disable a menu item with the <sref>aria-disabled</sref> attribute. If the menu item has its <pref>aria-haspopup</pref> attribute set to <code>true</code>, it indicates that the menu item may be used to launch a sub-level menu, and authors SHOULD display a new sub-level menu when the menu item is activated.</p>
-				<p>Authors MUST ensure that menu items are <a>owned</a> by an element with role <rref>menu</rref> or <rref>menubar</rref> in order to identify that they are related <a>widgets</a>. Authors MAY separate menu items into sets by use of a <rref>separator</rref> or an element with an equivalent role from the native markup language.</p>
+				<p>Authors MUST ensure that menu items are <a>owned</a> by an element with role <rref>group</rref>, <rref>menu</rref> or <rref>menubar</rref> in order to identify that they are related <a>widgets</a>. Authors MAY separate menu items into sets by use of a <rref>separator</rref> or an element with an equivalent role from the native markup language.</p>
```

Comment:
This to me suggests that a `menuitem` inside a `group` is valid even if that `group` does not descend from a `menu`. Do we want to clarify that `menuitem` instances must be contained within a menu?